internals/webhooks/subnamespace_webhook.go

```diff
@@ -166,6 +170,10 @@ func (a *SubNamespaceAnnotator) Handle(ctx context.Context, req admission.Reques
 			if isValid, err := ValidateAllResourceQuotaParamsValid(sns); !isValid {
 				return admission.Denied(err.Error())
 			}
+		} else if utils.GetSnsResourcePooled(sns.Object) == "true" {
+			if isValid, err := ValidateUpperResourcepoolHasQuota(sns); !isValid {
+				return admission.Denied(err.Error())
+			}
```

Comment:
Why doesn't `ValidateAllResourceQuotaParamsValid` catch this instead of using a new function?